src/sdk/pynni/nni/metis_tuner/Regression_GP/OutlierDetection.py

```diff
@@ -52,32 +57,32 @@ def _outlierDetection_threaded(inputs):
                    "difference": abs(samples_y_aggregation[samples_idx] - mu) - (2.33 * sigma)}
     return outlier
 
+
 def outlierDetection_threaded(samples_x, samples_y_aggregation):
     '''
     Use Multi-thread to detect the outlier
     '''
     outliers = []
 
-    threads_inputs = [[samples_idx, samples_x, samples_y_aggregation]\
-                            for samples_idx in range(0, len(samples_x))]
+    threads_inputs = [[samples_idx, samples_x, samples_y_aggregation]
+                      for samples_idx in range(0, len(samples_x))]
     threads_pool = ThreadPool(min(4, len(threads_inputs)))
-    threads_results = threads_pool.map(_outlierDetection_threaded, threads_inputs)
+    threads_results = threads_pool.map(
+        _outlierDetection_threaded, threads_inputs)
     threads_pool.close()
     threads_pool.join()
 
     for threads_result in threads_results:
         if threads_result is not None:
             outliers.append(threads_result)
         else:
-            print("error here.")
+            sys.stderr.write("Error: threads_result is None.")
```

Comment:
Use `logger.error()`

Reply:
There is no logger in this file.  If we used logger here will get `E0602: Undefined variable 'logger' (undefined-variable)`.

Reply:
Then create one with `logging.getLogger(__name__)`

Reply:
I don't think create a logger in every file is good solution.

Reply:
That's how Python works...

Reply:
How about I roll back this change due to this PR only solve the docstring problem.

Reply:
I feel no urge on this problem.
`print` is okay to me.